docs/core/install/runtime.md

```diff
@@ -50,7 +50,7 @@ export PATH=$PATH:$HOME/dotnet
 
 ## Install with a package manager
 
-You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-managers.md).
+You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-manager-ubuntu-1910.md).
```

Comment:
The text leads you to expect a general  Linux doc, but the new link is for a specific ubuntu release.